diff -r -u opt/cactus/include/boost/config/stdlib/libstdcpp3.hpp ../cactus/include/boost/config/stdlib/libstdcpp3.hpp
--- opt/cactus/include/boost/config/stdlib/libstdcpp3.hpp	2013-11-29 12:40:18.000000000 +0100
+++ ../cactus/include/boost/config/stdlib/libstdcpp3.hpp	2013-12-12 12:09:24.876369310 +0100
@@ -33,7 +33,8 @@
 
 #ifdef __GLIBCXX__ // gcc 3.4 and greater:
 #  if defined(_GLIBCXX_HAVE_GTHR_DEFAULT) \
-        || defined(_GLIBCXX__PTHREADS)
+        || defined(_GLIBCXX__PTHREADS) \
+        || defined(_GLIBCXX_HAS_GTHREADS)
       //
       // If the std lib has thread support turned on, then turn it on in Boost
       // as well.  We do this because some gcc-3.4 std lib headers define _REENTANT
diff -r -u opt/cactus/include/boost/thread/xtime.hpp ../cactus/include/boost/thread/xtime.hpp
--- opt/cactus/include/boost/thread/xtime.hpp	2013-11-29 12:40:07.000000000 +0100
+++ ../cactus/include/boost/thread/xtime.hpp	2013-12-12 12:11:35.089708333 +0100
@@ -20,7 +20,7 @@
 
 enum xtime_clock_types
 {
-    TIME_UTC=1
+    TIME_UTC_=1
 //    TIME_TAI,
 //    TIME_MONOTONIC,
 //    TIME_PROCESS,
@@ -68,7 +68,7 @@
 
 inline int xtime_get(struct xtime* xtp, int clock_type)
 {
-    if (clock_type == TIME_UTC)
+    if (clock_type == TIME_UTC_)
     {
         *xtp=get_xtime(get_system_time());
         return clock_type;
