diff -Nur v02-07/cmake/ilcsoft_default_settings.cmake v02-07-mod/cmake/ilcsoft_default_settings.cmake
--- v02-07/cmake/ilcsoft_default_settings.cmake	2017-05-26 16:57:19.000000000 +0200
+++ v02-07-mod/cmake/ilcsoft_default_settings.cmake	2017-05-26 17:01:40.000000000 +0200
@@ -19,7 +19,7 @@
 INCLUDE( ilcsoft_default_enable_ctest )
 INCLUDE( ilcsoft_default_library_versioning )
 INCLUDE( ilcsoft_default_build_output_directories )
-INCLUDE( ilcsoft_default_rpath_settings )
+# INCLUDE( ilcsoft_default_rpath_settings )
 #INCLUDE( ilcsoft_build_32bit_compatible )
 
 
